### PR TITLE
Fix Theia loading inside iframe

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### Referenced issue:
https://github.com/theia-ide/theia/issues/5046

Fix loading Theia in the iframe launched with protected connection for Google Chrome

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts [hunk 1]
```diff
@@ -65,6 +66,13 @@ export class BrowserKeyboardLayoutProvider implements KeyboardLayoutProvider, Ke
         }
     }
 
+    private isAbleToGetLayoutMap(): boolean {
+        if (browser.isChrome && window.top !== window && window.location.protocol === 'https:') {
```
Comment (spoenemann) [inline packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts]: Does this problem apply only to Chrome? According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Keyboard/getLayoutMap), the Opera browser also supports this API.

Should we wrap the code with try-catch to be sure we get all corner cases and supporting browsers?

**Comment by AndrienkoAleksandr** [comment]
Closed on favor https://github.com/theia-ide/theia/pull/5054

**Comment by amiramw** [comment]
without this bug fix version 0.6.0 is broken. is there any plan to release a patch version?

**Comment by AndrienkoAleksandr** [comment]
+1 for patch version

**Comment by spoenemann** [comment]
> is there any plan to release a patch version?

CC @marcdumais-work @svenefftinge

**Comment by spoenemann** [comment]
Yes, I would expect that to work (note that the fix is in #5054, though).

**Comment by marcdumais-work** [comment]
Ok. I'll prepare the new branch. Any volunteer to have a look when I am done, before I go ahead and release? ETA for new branch: hopefully ~30 mins if I do not encounter issues, else soon after lunch.

**Comment by marcdumais-work** [comment]
New branch: https://github.com/theia-ide/theia/tree/0.6.1

**Comment by marcdumais-work** [comment]
The branch built and tests passed locally for me.

**Comment by marcdumais-work** [comment]
@AndrienkoAleksandr / @amiramw : Can someone validate that the `0.6.1` works ok, w-r to the issue at hand? I can then release it tomorrow.

**Comment by AndrienkoAleksandr** [comment]
@marcdumais OK, I will test this version today.

**Comment by AndrienkoAleksandr** [comment]
Hello, @marcdumais I tested 0.6.1 locally and inside che-theia on the openshift.io and minikube. Works good for me.

**Comment by marcdumais-work** [comment]
@AndrienkoAleksandr Great, thanks for confirming :) 

I'll go ahead and attempt to release `0.6.1`. It's the first time I publish a bugfix release like this - hopefully it goes smooth!

**Comment by marcdumais-work** [comment]
Release done.

**Pull request closed without merging**